plugins/trader/portfolioManager.js

```diff
@@ -208,8 +225,9 @@ Manager.prototype.buy = function(amount, price) {
     'Attempting to BUY',
     amount,
     this.asset,
-    'at',
-    this.exchange.name,
```

Comment:
no longer necessary as logPortfolio is now being executed when trade advice is given which includes the name of the exchange.